### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by SweeXordious**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,45 +3,64 @@ Web3j Open API
...
+#### Interact with smart contracts without code:
+Being able to generate an OpenAPI project from a smart contract and interacting with it using HTTP requests, eliminates the need to code any interactions and being able to send them to the blockchain. Thus, it is legitimate to say that the Web3j-OpenAPI lets you interact with the Ethereum blockchain using no code, besides solidity.
+
 
 ## Use cases
+An OpenAPI project can be generated using the following command:
 
 ```ssh
-$ epirus generate openapi Greeter.sol -p com.helloworld -o /home/rachid/helloworld
+$ epirus generate openapi -p com.helloworld -o . --abi helloworld.abi --bin helloworld.bin --name helloworld
```
Comment (iikirilov) [inline README.md:31]: this part of the documentation needs to go into epirus-cli. 

you can link to epirus-cli here instead

Comment (SweeXordious) [reply README.md:31]: what about now ?

File: README.md [hunk 2]
```diff
@@ -3,45 +3,64 @@ Web3j Open API
...
 ```ssh
-$ ./gradle shadowJar // Create an executable JAR that starts the server
-$ java -jar helloworld-0.1.0-all.jar
+$ ./gradlew shadowJar // Create an executable JAR that starts the server
+$ java -jar helloworld-server-all.jar
 ...
 ```
 
-Or using the CLI:
+**Using the CLI:**
```
Comment (iikirilov) [inline README.md:50]:
```suggestion
**Starting the server using the CLI:**
```

File: README.md [hunk 3]
```diff
@@ -3,61 +3,15 @@ Web3j Open API
...
-val service = ClientService("http://localhost:8080")
-val helloWorld = ClientBuilder.build(HelloWorldApi::class.java, service)
-
-val receipt = helloWorldApi.contracts.greeter.deploy(
-    GreeterDeployParameters("Test greeter")
-)
-
-val greeter = helloWorldApi.contracts.greeter.load(receipt.contractAddress)
-```
+**For more explanations**, check the documentation, this blog post, or the demo project.
```
Comment (SweeXordious) [inline README.md:17]:
```suggestion
**For more explanations**, a well detailed documentation and a sample project are on the way.
```

File: README.md [hunk 4]
```diff
@@ -3,61 +3,15 @@ Web3j Open API
...
- * helloworld:
-   * openapi
-   * server: web3j-openapi-server (org.web3j.openapi.server.MainKt or generated main)
+## Description 
+Web3j-OpenAPI provides a way to interact with the Ethereum blockchain via simple and intuitive HTTP requests, abstracting the coding layer. These interactions can be done using plain HTTP requests or via the Swagger-UI, which is generated with every project.
 
-## Use cases
+The workflow can be summed in the following steps:
+- Writing a solidity smart contract
+- Generating the corresponding OpenAPI project using Web3j-OpenAPI
```
Comment (SweeXordious) [inline README.md:13]:
```suggestion
- Generating the corresponding **OpenAPI** project using **Web3j-OpenAPI**
```

**Comment by AlexandrouR** [comment]
LGTM

**Pull request closed without merging**